menu/server_configuration.sh

```diff
@@ -71,6 +81,7 @@ https://www.techandme.se/open-port-80-443/"
             then
                 bash $SCRIPTS/activate-tls.sh
             else
+                print_text_in_color "$ICyan" "Downloading the activate-tls script..."
```

Comment:
The Let's Encrypt script I think would be better.

Reply:
fixed